src/qt/test/test_main.cpp

```diff
@@ -70,7 +70,7 @@ int main(int argc, char* argv[])
     #if defined(WIN32)
         if (getenv("QT_QPA_PLATFORM") == nullptr) _putenv_s("QT_QPA_PLATFORM", "minimal");
     #else
-        setenv("QT_QPA_PLATFORM", "minimal", /* overwrite */ 0);
+        setenv("QT_QPA_PLATFORM", "minimal", 0 /* overwrite */);
```

Comment:
I think for consistency named args that are not enforced should be trailing. If it was kept in the beginning, it could confuse devs and trick them into adding an `=`, which again would fail clang-tidy. See also the comment in `src/test/fuzz/tx_pool.cpp`, which I moved to the end.

Reply:
maybe `/* don't overwrite */` ?

Reply:
It is the correct argument name according to `man`, but the headers shipped on the system may pick a different name (implementation detail).

```
$ man setenv | head -9
SETENV(3)             Linux Programmer's Manual            SETENV(3)

NAME
       setenv - change or add an environment variable

SYNOPSIS
       #include <stdlib.h>

       int setenv(const char *name, const char *value, int overwrite);